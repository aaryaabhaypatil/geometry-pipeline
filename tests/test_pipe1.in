-26.20, 4.42
-13.00, 10.39
