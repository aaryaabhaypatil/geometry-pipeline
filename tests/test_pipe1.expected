read 2 points
-26.20, 4.42
-13.00, 10.39
This is not a triangle
